src/load_balancing.hpp

```diff
@@ -43,6 +43,7 @@ typedef enum CassBalancingState_ {
 typedef enum CassHostDistance_ {
   CASS_HOST_DISTANCE_LOCAL,
   CASS_HOST_DISTANCE_REMOTE,
+  CASS_HOST_DISTANCE_REMOTE2,
```

Comment:
LOCAL is same rack and same dc, REMOTE is same dc and REMOTE2 is really remote right? Shouldn't we use something more descriptive as name? Or at least add some comment here so we know in the future?

Reply:
Good point, thanks!

It depends on which policy you use:
* `DCAwarePolicy` uses `LOCAL` for same DC, `REMOTE` for different DC.
* `RackAwarePolicy` uses `LOCAL` for same rack, `REMOTE` for same DC, different rack, `REMOTE2` for different DC.
* other policies always use `LOCAL`.

I have added a code comment.